testutil/ibc_testing/generic_setup.go

```diff
@@ -3,21 +3,43 @@ package ibc_testing
 import (
 	"testing"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	ibctesting "github.com/cosmos/ibc-go/v3/testing"
-	"github.com/cosmos/interchain-security/testutil/e2e"
+	e2eutil "github.com/cosmos/interchain-security/testutil/e2e"
+	consumerkeeper "github.com/cosmos/interchain-security/x/ccv/consumer/keeper"
 )
 
 // Contains generic setup code for running e2e tests against a provider, consumer,
 // and/or democracy consumer app.go implementation. You should not need to modify or replicate this file
 // to run e2e tests against your app.go implementations!
 
 var (
+	FirstConsumerChainID = ibctesting.GetChainID(2)
 	provChainID          = ibctesting.GetChainID(1)
-	democConsumerChainID = ibctesting.GetChainID(2)
+	democConsumerChainID = ibctesting.GetChainID(5000)
 )
 
+// ConsumerBundle serves as a way to store useful in-mem consumer app chain state
+// and relevant IBC paths in the context of CCV e2e testing.
+type ConsumerBundle struct {
```

Comment:
Lmk if there's anyway to make this abstraction more generic, or named in a better way. It's pretty much "all the state associated with a consumer that you need for e2e tests"